app/uk/gov/hmrc/emcstfe/models/createMovement/EadEsadDraftModel.scala

```diff
@@ -38,7 +38,7 @@ case class EadEsadDraftModel(
     <urn:OriginTypeCode>{originTypeCode.toString}</urn:OriginTypeCode>
     <urn:DateOfDispatch>{dateOfDispatch}</urn:DateOfDispatch>
     {timeOfDispatch.mapNodeSeq(value => <urn:TimeOfDispatch>{value}</urn:TimeOfDispatch>)}
-    {importSad.mapNodeSeq(_.map(_.toXml))}
+    {importCustomsDeclaration.mapNodeSeq(_.map(_.toXml))}
   </urn:EadEsadDraft>
 }
 
```

Comment:
Should we rename this file? due to `sad` no longer there. But then we have element that still has `sad` i.e. `<urn:EadEsadDraft>`

In `ImportCustomsDeclarationModel` class below, urn:element is updated so maybe we can update this one as well